misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobConsumer.kt

```diff
@@ -118,7 +125,29 @@ internal class SqsJobConsumer @Inject internal constructor(
           } else {
             true
           }
+        }.size
+
+    }
+
+    private fun balancedMaxReceivers(): Int {
+      // Read the max from the per pod flag.
+      val maxPerPod = receiversPerPodForQueue()
```

Comment:
slight nit - I think this is a _slightly_ different meaning than it's previous interpretation.

I wouldn't be opposed to introducing a third config flag for "max per queue" that is read if and only if we're using `BALANCED_MAX` as a config value.

Reply:
added